internal/driver/service_test.go

```diff
@@ -44,17 +46,15 @@ func (s *MockSDKService) Devices() []contract.Device {
 
 func (s *MockSDKService) AddDiscoveredDevices(discovered []dsModels.DiscoveredDevice) {
 	for _, d := range discovered {
-		_, _ = s.AddDevice(contract.Device{
-			Name:      d.Name,
-			Protocols: d.Protocols,
-			Profile: contract.DeviceProfile{
-				Name: "LLRP.Device.Profile",
-			},
+		_, _ = s.AddDevice(models.Device{
+			Name:        d.Name,
+			Protocols:   d.Protocols,
+			ProfileName: "LLRP.Device.Profile",
```

Comment:
I think you changed the name of this, right?

Reply:
I have changed this